src/lib.rs

```diff
@@ -664,12 +664,33 @@ impl<'a> TryFrom<&'a str> for Crontab {
 #[derive(Debug, Copy, Clone)]
```

Comment:
@Shresth72 thanks for your contribution and verify.

I mean to move all the examples here. Because people who goes to the struct doc page may not find the constructor methods first but always read the struct document.

If a user already find the impl From, he/she already know how to construct it and the example provide less value than adding it here.